tests/django_test_setup.py

```diff
@@ -1,8 +1,16 @@
+from pathlib import Path
+from typing import Dict, Optional
+
 import django
 from django.conf import settings
 
-if not settings.configured:
+
+def setup_test_config(components: Optional[Dict] = None):
```

Comment:
Instead of configuring Django setting when we import `django_test_setup`, it's now wrapped in a function, so we can configure the settings.